### PR TITLE
fix security header for header

### PR DESCRIPTION
**Pull request by jeremy-babylonchain**

### CODE REVIEW

**Comment by jrwbabylonchain** [comment]
1. build failed
2. I think this is a hotfix PR. so should point to main branch.

File: internal/api/middlewares/security.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ func SecurityHeadersMiddleware() func(http.Handler) http.Handler {
 		FrameDeny:             true, // Equivalent to X-Frame-Options: DENY
 		ContentTypeNosniff:    true, // Equivalent to X-Content-Type-Options: nosniff
 		BrowserXssFilter:      true, // Equivalent to X-XSS-Protection: 1; mode=block
-		ContentSecurityPolicy: "default-src 'self'; script-src 'self'; style-src 'self'; img-src 'self'; font-src 'self'; object-src 'none'; frame-ancestors 'self'; form-action 'self'; block-all-mixed-content; base-uri 'self';",
-		ReferrerPolicy:        "strict-origin-when-cross-origin", // Setting Referrer-Policy
+		ContentSecurityPolicy: "default-src 'self'; script-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com https://stackpath.bootstrapcdn.com; style-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com https://stackpath.bootstrapcdn.com; img-src 'self' data: https://cdnjs.cloudflare.com https://stackpath.bootstrapcdn.com; font-src 'self' https://cdnjs.cloudflare.com https://stackpath.bootstrapcdn.com; object-src 'none'; frame-ancestors 'self'; form-action 'self'; block-all-mixed-content; base-uri 'self';",
```
Comment (jrwbabylonchain) [inline internal/api/middlewares/security.go:16]: question: @jeremy-babylonchain could u help me understand why this url need to be added? i mean, why this url?

Comment (jeremy-babylonchain) [reply internal/api/middlewares/security.go:16]: https://cdnjs.com/libraries/swagger-ui
the imgs and styles and css are delivered from these cdn

Comment (jrwbabylonchain) [reply internal/api/middlewares/security.go:16]: gotcha. how about another approach? 
we check the url path of the request. if it's calling `/swagger` path, then we attach the security policy header with the `unsafe-inline`?

**Comment by jrwbabylonchain** [comment]
@jeremy-babylonchain let's merge into main based on the new release strategy where all bug fixes shall go directly into main branch and to be released.

**Review by jrwbabylonchain — APPROVED** [review]

**Pull request merged**